### ISSUE TITLE
Restrict PICO CSS to certain part of HTML 

### ISSUE THREAD

**Issue by mrniamster** [picocss/pico#378]
Please search for duplicate or closed issues first.
  
## Describe the issue
  Currently picocss is interfering with the default devtools tools provided by frameworks like react-query.
### Current Behavior
A concise description of the bug.
I Would like to have a varablt say $parent-selector:"#___nuxt" , so i can tell picocss to load till certain page of the body
### Expected Behavior
A concise description of what you expected.
The devtools are getting affected 
![image](https://github.com/picocss/pico/assets/52563039/bcc66303-d7af-470c-8636-f2d051c85fd1)

### Reproduction URL
We recommend including a link to a minimal reproduction of the bug using CodePen or a similar tool.
**Please do not link to your actual project.** Instead, we need a reduced test case in a new project without any unnecessary code.

### Environment
Example: OS, versions, browser details.
- Operating System: Windows_NT
- Node Version:     v18.11.0
- Nuxt Version:     3.5.3
- Nitro Version:    2.4.1
- Package Manager:  npm@9.6.5
- Builder:          vite
- User Config:      routeRules, nitro, css, vite, postcss, app, modules, content, googleSignIn, runtimeConfig     
- Runtime Modules:  nuxt-vue3-google-signin@0.0.8, @pinia/nuxt@0.4.11, @nuxt/devtools@0.6.1, @nuxt/content@2.7.0, 
@pinia/nuxt@0.4.11
- Build Modules:    -

  "@picocss/pico": "^1.5.10",
    "@pinia/nuxt": "^0.4.11",

**Comment by mrniamster** [comment]
@lucaslarroche  Can you  please look into this.

**Comment by rsuurd** [comment]
I think the same applies to in my case a google maps map. The markers get assigned a role=[button] attribute once they have been made clickable which results in the markers getting the pico style, making them blue squares.

![Scherm­afbeelding 2023-06-22 om 13 49 40](https://github.com/picocss/pico/assets/361016/c0b4608a-7144-43b6-8f88-d9b5df22b195)

Is there a workaround for this?

**Comment by mrniamster** [comment]
@rsuurd agreed,  I had to move from PICO due to this but would really appreciate if this gets resolved

**Comment by nakibrayan3** [comment]
you can import only the needed modules using `scss`.

check out the docs: https://picocss.com/docs/customization.html, under section `Importing Pico SASS library`

**Comment by mrniamster** [comment]
@nakibrayan3  But how does it solves the given requirement question of this issue, it still applies the styles to "non-required" componenets like devtools bar floating widget,

**Comment by nakibrayan3** [comment]
i don't really know how to fix your problem, sorry.

**Comment by cemrehancavdar** [comment]
Restricting pico can be possible if you add all scss rules :not(pico-disable *) one by one imho and add .pico-disable to elements.
But you can also build yourself a custom pico with .pico contains all layout part in scss/pico.scss this will make pico work with .pico only. So this is a trade of but it is possible. 

Result css will be:
[css.zip](https://github.com/picocss/pico/files/11884523/css.zip)

This is the way i use it with leaflet.

**Comment by VargoGames** [comment]
is it possible to have a div with some class applied that acts like reset.css, so that none of pico stuff is applied?

**Comment by lucaslarroche** [comment]
Pico v2 will offer a CSS version that only enables Pico styles for specified parts of your HTML code.

**Comment by mrniamster** [comment]
Thank you Lucas, i was still booking for update on this !! SHould be great

**Comment by marksteven** [comment]
having same issue with Pico overriding Vidstack video player buttons. Will look into the suggestion of Pico V2

**Comment by blachawk** [comment]
I am also facing this issue and did not know this was a limiation.  How do we set a parent ID on areas that should only have pico CSS . for example, with _basics.scss being applied to input elements...

go from this :
```css
input, select, textarea {...}
```
 to this :

```css
#myArea input, #myArea select, #myArea textarea {...}
```

**Comment by mrniamster** [comment]
Hi Team, Thanks for implementing the conditional PICO Version, 

One problem  @warren-bank @lucaslarroche 

It still uses the ".grid" class and affect  non pico , and pico at same time 
As you can see it applies grid from picocss even if its unstyled one, (currently its affecting my other sites.)
![image](https://github.com/user-attachments/assets/95953917-3ed7-41fe-8286-59e757bc797b)

**Comment by y2kbugger** [comment]
is there an opposite to this for standard version, e.g. to _opt out_ for just one div
